src/borg/testsuite/archiver.py

```diff
@@ -3529,6 +3529,13 @@ def test_strip_components_doesnt_leak(self):
                 res = self.cmd('extract', "--debug", self.repository_location + '::test', '--strip-components', '0')
                 self.assert_true(marker not in res)
 
+    def test_do_not_mention_archive_if_you_can_not_find_repo(self):
+        """https://github.com/borgbackup/borg/issues/6014"""
+        archive = self.repository_location + '-this-repository-does-not-exist' + '::test'
+        output = self.cmd(*['info', archive], exit_code=2, fork=True)
```

Comment:
Wondering about `*['info', archive]` - isn't that construction the same as writing `'info', archive` there?

Why did you use it?

Reply:
Oops. I'll fix it. Thanks!